modules/optflow/src/learn_prior.py

```diff
@@ -0,0 +1,123 @@
+import os, sys
```

Comment:
An example of how to use this script, so that anyone might be able to reproduce the results would be great.  Include where/how to get the data, and sequence of steps necessary to construct the prior file.